pynumaflow/function/server.py

```diff
@@ -72,40 +47,30 @@ class UserDefinedFunctionServicer(udfunction_pb2_grpc.UserDefinedFunctionService
     Example invocation:
     >>> from typing import Iterator
     >>> from pynumaflow.function import Messages, Message, MessageTs, MessageT, \
-    ...     Datum, Metadata, UserDefinedFunctionServicer
-    ... import aiorun
+    ...     Datum, Metadata, Server
     ...
     >>> def map_handler(key: [str], datum: Datum) -> Messages:
     ...   val = datum.value
     ...   _ = datum.event_time
     ...   _ = datum.watermark
-    ...   messages = Messages(Message.to_vtx(key, val))
+    ...   messages = Messages(Message(val, keys=keys))
     ...   return messages
     ...
+    ... def reduce_handler(key: str, datums: Iterator[Datum], md: Metadata) -> Messages:
+    ...           "Not supported"
```

Comment:
We can just mention in the docstring instead that reduce handler is not supported yet